flink-python/pyflink/table/statement_set.py

```diff
@@ -0,0 +1,93 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+from abc import ABCMeta
+
+from pyflink.util.utils import to_j_explain_detail_arr
+
+__all__ = ['StatementSet']
+
+
+class StatementSet(object):
+    """
+    A StatementSet accepts DML statements or Tables,
+    the planner can optimize all added statements and Tables together
+    and then submit as one job.
+
+    .. note::
+    The added statements and Tables will be cleared
+    when calling the `execute` method.
+    """
+
+    __metaclass__ = ABCMeta
```

Comment:
It seems we don't need this. The class is not an abstract class in the Python side.